frontend/src/modules/organization/components/details/organization-details-contributors.vue

```diff
@@ -6,6 +6,7 @@
     :search-config="memberSearchFilter"
     :saved-views-config="memberSavedViews"
     :custom-config="customAttributesFilter"
+    :exclude-filters="excludeFilters"
```

Comment:
Can you revert all logic added to Filter.vue and FilterItem.vue?
You don't need to add the `exclude-filters` for it to work.

What's happening is that whenever you open an organization profile, a "segmentId" parameter is being added to the url.
Then on "Filter.vue", by default we parse all the query params on this component to parse them as filters. However, this segmentId, along with projectGroup shouldn't be accounted for.
I believe the fix can simply be achieved by updating:
"  const {
    projectGroup, segmentId, menu, ...parsedQuery
  } = query;"
  
  Adding "segmentId" in line 193 of Filters.vue

Reply:
Got it

Reply:
done